charts/internal/cilium/charts/agent/templates/daemonset.yaml

```diff
@@ -288,6 +288,26 @@ spec:
           name: cilium-run
         resources:
           {{- toYaml .Values.monitor.resources | trim | nindent 10 }}
+{{- end }}
+      # Masquerade traffic to upstream DNS server
+{{- if .Values.global.snatToUpstreamDNS.enabled }}
+      - name: add-snat-rule-to-upstream-dns
+        image: {{ index .Values.global.images "alpine-iptables" }}
+        imagePullPolicy: IfNotPresent
+        securityContext:
+          privileged: true
```

Comment:
Why does this container need to be privileged? It should be possible to supply it with just the required capabilities.